diesel_cli/README.md

```diff
@@ -4,6 +4,34 @@ Diesel CLI
 Diesel CLI is a tool that aids in managing your database schema. Migrations are
 bi-directional changes to your database that get applied sequentially.
 
+Installation
+---------------
+
+The diesel cli, by default, requires [`libpq`][postgres], [`sqlite`][sqlite]
+and [`mysql`][mysql], some of which may already be included on your system. See
+which dependencies are already present with:
+
+```sh
+which psql
+which sqlite3
+which mysql
```

Comment:
I don't think this will work for linux users, where `mysql` the binary is included in a separate package from `libmysql` the object we need to link against.